cli/kaos_cli/commands/build.py

```diff
@@ -76,6 +76,17 @@ def build(backend: BackendFacade, cloud, env, force, verbose, yes, local_backend
             click.style("Info", bold=True, fg='green'),
             click.style(cloud, bold=True, fg='red')))
 
+    # validate unused port for DOCKER
+    if cloud == DOCKER:
+        is_available = validate_unused_port(80)
+        if not is_available:
+            click.echo(
+                "{} - Network port {} is used but is needed for building {} backend in {}".format(
+                    click.style("Warning", bold=True, fg='yellow'),
+                    click.style("80", bold=True),
+                    click.style("kaos", bold=True),
+                    click.style(cloud, bold=True, fg='red')))
+    
```

Comment:
missing a `sys.exit(1)` since the check doesn't stop the user...

```
$ kaos build -c DOCKER -vf

Warning - Performing force build of the backend
Warning - Are you sure about building kaos backend in DOCKER? [y/N]: y
Warning - Network port 80 is used but is needed for building kaos backend in DOCKER
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "null" (hashicorp/null) 2.1.2...
- Downloading plugin for provider "external" (hashicorp/external) 1.2.0...
- Downloading plugin for provider "kubernetes" (hashicorp/kubernetes) 1.9.0...
^C
Aborted!
```

Reply:
I shall add the `sys.exit(1)` there. The reason I didn't add it is I assumed that we're just raising a warning as per the issue spec.

Reply:
Done.